amphibia-reptilia.csl

```diff
@@ -42,11 +42,16 @@
   </macro>
   <macro name="editor">
     <names variable="editor">
-      <name delimiter-precedes-last="always" initialize-with="." name-as-sort-order="all"/>
-      <label form="short" text-case="capitalize-first" prefix=", "/>
+      <name delimiter-precedes-et-al="never" delimiter-precedes-last="never" et-al-min="19" et-al-use-first="19" et-al-subsequent-min="19" et-al-subsequent-use-first="19" initialize-with="." name-as-sort-order="all"/>
+      <label form="short" text-case="capitalize-first" prefix=", " suffix=", "/>
     </names>
   </macro>
-  <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" collapse="year">
+  <macro name="book_author">
+    <names variable="container-author">
+      <name suffix=", " delimiter-precedes-et-al="never" delimiter-precedes-last="never"/>
+    </names>
+  </macro>
+  <citation name-form="short" and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never" et-al-min="4" et-al-use-first="1" et-al-subsequent-min="4" et-al-subsequent-use-first="1" disambiguate-add-year-suffix="true" collapse="year">
```

Comment:
I use that in every names definition to avoid unwanted behaviour before I discovered that there were overal definitions in <citation> and <bibliography> that I could set as well. I'll take it out.